lib/main.py

```diff
@@ -129,7 +129,7 @@ def background_thread_task():
             try:
                 log(nc, LogLvl.ERROR, str(e))
                 nc.providers.task_processing.report_result(task["id"], error_message=str(e))
-            except (NextcloudException, httpx.RequestError) as net_err:
+            except (NextcloudException, HTTPError) as net_err:
```

Comment:
Hey,

I just found out that based  on https://github.com/jawah/niquests/blob/main/src/niquests/exceptions.py#L19 we may need to catch niquests.RequestException instead of just HTTPError to cover more error cases. Could you change this?